mock-server/data/info.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
May be needed update rules for camelCase in eslint-config? 

https://eslint.org/docs/rules/camelcase

For example: 
  "rules": {
    "camelcase": [
      2,
      {
        "properties": "never"
      }
    ]
}

Reply:
nice point, created issue with some linter improvements https://github.com/uploadcare/uploadcare-upload-client/issues/284
Do you have any other suggestions?